libtransmission/session.h

```diff
@@ -975,6 +975,18 @@ struct tr_session
         return settings().encryption_mode;
     }
 
+    [[nodiscard]] auto serialize_encryption_mode() const noexcept
+    {
+        auto var = libtransmission::serializer::Converters::serialize(settings().encryption_mode);
+        TR_ASSERT(var.has_value());
+        return var;
+    }
+
+    bool deserialize_encryption_mode(tr_variant const& var) noexcept
+    {
+        return libtransmission::serializer::Converters::deserialize(var, &settings_.encryption_mode);
```

Comment:
Use `serializer::to_value()` instead; it's a little terser + clearer

Reply:
Sorry, my mistake. I'm traveling & am reviewing on mobile. Your original version was fine. 🫠

Reply:
Let me drop that commit. 👌🏻